packages/@react-spectrum/table/stories/TableDnDExamples.tsx

```diff
@@ -543,7 +544,13 @@ export function DragBetweenTablesRootOnlyExample(props) {
       </Flex>
       <Flex direction="column" margin="size-100">
         <Text alignSelf="center">Table 2</Text>
-        <TableView aria-label="Second TableView" selectionMode="multiple" width={400} dragAndDropHooks={dragAndDropHooksSecond} {...tableViewProps}>
+        <TableView
```

Comment:
If I remove all the items from table two, the table body height is about 10 pixels and I see some grey lines in the table, which when inspected reveals that they are an illustrated message. Should this example have a min height?